sources/packages/web/src/components/generic/formio.vue

```diff
@@ -11,7 +11,7 @@ import ApiClient from "../../services/http/ApiClient";
 import FormUploadService from "../../services/FormUploadService";
 
 export default {
-  emits: ["submitted", "loaded", "changed"],
+  emits: ["submitted", "loaded", "changed", "custom"],
```

Comment:
can we change `custom` to something meaningful? like `customeCallback` or to something that makes more sense

Reply:
All 4 in the list are callbacks actually, I would prefer "customEvent" over using the word callback here. 
What is your opinion?

Reply:
> All 4 in the list are events actually, I would prefer "customEvent" over using the word callback here.
> What is your opinion?

I agree with @dheepak-aot. Callback would be the methods that would be associated to these events. I would prefer to leave it as is.

Reply:
Changed from custom to customEvent